### DISCUSSION TITLE
Newest Intel Wifi kext file problem

### DISCUSSION THREAD

**Discussion by PPPPPH** [macOS-Dell-Latitude-E7470/macOS-Dell-Latitude-E7470#26] [Q&A] [answered]
looks like the current AirportItlwm.kext has been updated in your recently update to opencore 0.66,but I reinstall the whole macos and WiFi is still not usable. check the md5 and find out the file is not official AirportItlwm_v1.2.0_stable nor latest debug 1.3.0 .
Copy the official AirportItlwm_v1.2.0_stable fix the WiFi.
BTW,If I want to add nvmefix.kext,all I need to do is paste nvmefix.kext into exist EFI/OC/kext folder and run propertree,select config.plist and make a OC snapshot,is that correct?

**Comment by adityabakare** [comment, marked as answer]
WiFi works fine with the one I've included. I've use the debug alpha which was built up to [this commit](https://github.com/OpenIntelWireless/itlwm/commit/ecf78fcf28b985df1a7d669a3f2e558ff7ada3af). You might have checked the md5 with the newer alpha. 
And yes, for adding NVMeFix, the method that you've mentioned is correct. I haven't included it because I don't have an NVMe SSD.

Converting this to a Discussion.

**Reply by PPPPPH** [reply]
My wireless card is 3165, use stable kext solve the problem.
If I want to install 10.15(for enable sidecar on old iPad, the hack cant be done in Big Sur), just replace the kext with 10.15 version and make a OC snapshot then all good? Anything I need to change in config.plst?
I also want to change my wireless card to a BCM one for better experience, Besides add&delete some kext and make a OC snapshot,anything else I need to do with config.plist?
Thanks for your work and answering. New to either Open core or Github, still confused for the whole process.

**Reply by adityabakare** [reply]
Technically yes. Just replace the Big Sur kext with Catalina one and it should boot fine. But, I won't be able to provide any help regarding the Catalina installation.
And if you want to use a Broadcom card, remove the AirportItlwm kext extirely, and use the ones specific to the card. And use ProperTree to make the changes to the plist.
